packages/ca_certificates.rb

```diff
@@ -45,13 +57,18 @@ def self.install
     FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/etc/ssl/certs/")
     FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/bin")
     FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/share/ca-certificates/")
-    system "touch #{CREW_DEST_PREFIX}/etc/ca-certificates.conf"
     system "make DESTDIR=#{CREW_DEST_DIR} install"
-    FileUtils.ln_s Dir.glob("#{CREW_PREFIX}/share/ca-certificates/mozilla/*"),
-                   "#{CREW_DEST_PREFIX}/etc/ssl/certs/"
-  end
-
-  def self.postinstall
-    system 'update-ca-certificates --hooksdir "" --fresh'
+    @date_temp = `date -u`
```

Comment:
You could chomp here too.  Then every variable reference doesn't need it.
```
@date_temp = `date -v`.chomp
```

Reply:
Nice.

Reply:
Whoops. Just realized you should have put ```date-u``` here.